### PR TITLE
[MIG] shopinvader_wishlist: Migration to 14.0

### PR DESCRIPTION
**Pull request by chafique-delli**

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]
Code LGTM

**Comment by rousseldenis** [comment]
@chafique-delli Could you put on your PR description when it depends on a non migrated module (with the PR link) ? It's easier for reviewers. Thanks

File: oca_dependencies.txt [hunk 1]
```diff
@@ -1 +1 @@
-# See https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#oca_dependencies-txt
+sale-workflow
```
Comment (simahawk) [inline oca_dependencies.txt:1]: we don't need this we use MQT=PIP

File: shopinvader_wishlist/models/product_set.py [hunk 2]
```diff
@@ -22,15 +26,26 @@ class ProductSet(models.Model):
     )
 
     def get_line_by_product(self, product_id=None, invader_variant_id=None):
-        if not product_id and not invader_variant_id:
+        # Backward compat
```
Comment (simahawk) [inline shopinvader_wishlist/models/product_set.py:29]: this should be dropped

File: shopinvader_wishlist/services/wishlist.py [hunk 3]
```diff
@@ -61,6 +60,16 @@ def add_to_cart(self, _id):
         # return new cart
         return cart_service._to_json(cart)
 
+    def add_items_to_cart(self, _id, **params):
```
Comment (simahawk) [inline shopinvader_wishlist/services/wishlist.py:63]: there are other changes to include

**Review by simahawk — CHANGES_REQUESTED** [review]

**Comment by simahawk** [comment]
I opened https://github.com/shopinvader/odoo-shopinvader/pull/972 to include all recent fix/imp and remove deprecated code which requires #970

**Pull request closed without merging**